Remove service name and navigation from Header component (#6423)
alphagov/govuk-frontend: CHANGELOG.md

We introduced this change in [pull request #6382: Remove deprecated `govuk-pagination__item--ellipses` class](https://github.com/alphagov/govuk-frontend/pull/6382).

#### Update the class on the GOV.UK logo link in the Header component

The value of the `class` attribute on the link to the GOV.UK homepage has been updated to `govuk-header__homepage-link`.

> If you're not using Nunjucks macros, you will need to update this class manually.

Reviewer: ```suggestion
If you're not using Nunjucks macros, you'll need to update this class manually.
```

```html
<!-- Before -->
<a href="//gov.uk" class="govuk-header__link govuk-header__link--homepage">
  [...]
</a>

<!-- After -->
<a href="//gov.uk" class="govuk-header__homepage-link">
  [...]
</a>
```

This change was introduced in [pull request #6423: Remove service name and navigation from Header component](https://github.com/alphagov/govuk-frontend/pull/6423)

#### Stop including the service name and navigation items in the Header component

We deprecated including the service name and navigation in the Header in [GOV.UK Frontend 5.9.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.9.0).

> This functionality has now been removed.

Reviewer: ```suggestion
We've now removed this functionality.
```
